### ISSUE TITLE
8.2.16. New - Screen Name/Title: Trial a service - Missing Visible Label for Email Content Field

### ISSUE THREAD

**Issue by jzbahrai** [cds-snc/notification-planning#2462]
Component: Create reusable template - Email Content Field
Observation:
Element: <label class="form-label sr-only" for="template_content">Email content</label>
The label is visually hidden, and there's no other visible text label. Relying only on aria-required and screen reader-only labels excludes sighted keyboard users.

(Attachment: 8.2.16. Create reusable template - Email Content Field.png)

Recommendation: 
Make the label visible or associate it clearly with a visible heading or instruction (e.g., reuse “Email message” legend as a visible label).
